### ISSUE TITLE
Comparison experiment using football dataset

### ISSUE THREAD

**Issue by miguelgfierro** [Azure/fast_retraining#21]
- [x] Download dataset and put it on fileshare
- [x] Create data loader
- [x] Initial preprocessing
- [x] Create initial xgboost pipeline
- [x] Create initial lightgbm pipeline
- [x] Create comparison on test set
- [x] Find concept drift
